src/api/helpers/encrypt/generate-initialisation-vector/index.ts

```diff
@@ -0,0 +1,13 @@
+import crypto from 'crypto';
+import { FINANCIAL_DETAILS } from '../../../constants';
+
+const { IV } = FINANCIAL_DETAILS.ENCRYPTION;
+
+/**
+ * generateInitialisationVector
+ * generates initialisation vector for encrypting data
+ * @returns {String} initialisation vector
+ */
+const generateInitialisationVector = () => crypto.randomBytes(IV.BYTES).toString(IV.ENCODING).slice(0, 16);
```

Comment:
Do we need the slice?

If so, can we update the documentation and maybe could use a constant for the values

Reply:
the slice is needed to increase to 16 bytes, or else it will be 12 bytes